code/style.css

```diff
@@ -1,4 +1,310 @@
+}
+
+.container-two img {
+  display: flex;
+  border-radius: 5px;
+}
+
+.text-block {
+  position: absolute;
+  bottom: 10px;
+  background-color: rgba(212, 83, 94, 0.692);
+  font-size: 25px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+/*Footer*/
+footer {
+  background: rgb(212, 83, 93);
+  width: 100%;
+  height: 150px;
+}
+
+h4 {
+  font-size: 20px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  padding: 40px;
+}
+
+
+/*** MEDIA QUERIES ***/
+
+/*** TABLET ***/
+@media (min-width: 668px) and (max-width: 1023px) {
+
+  body {
+    min-width: 720px;
+  }
+
+  /*Header*/
+  header {
+    width: auto;
+    background: rgba(247, 222, 0, 0.959);
+    height: 150px;
+    display: flex;
+    align-items: center;
+  }
+
+  nav {
+    display: flex;
+    display: inline-block;
+    margin-left: 90px;
+    margin-right: 20px;
+  }
+
+  nav a {
+    margin: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    text-align: end;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  /*Big News*/
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .container img {
+    width: 630px;
+    border-radius: 5px;
+  }
+
+  /*Other News*/
+  .other-news {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .container-two {
+    position: relative;
+    width: 43%;
+    margin: 12px;
+    border-radius: 5px;
+    box-shadow: 5px 5px 0 0 rgba(207, 73, 84, 0.705);
+  }
+
+  .container-two img {
+    height: 450px;
+    border-radius: 5px;
+  }
+
+  .text-block {
+    position: absolute;
+    bottom: 10px;
+    background-color: rgba(212, 83, 94, 0.692);
+    color: white;
+    font-size: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+}
+
+/* COMPUTER */
+@media (min-width: 1024px) {
+
+  body {
+    min-width: 1060px;
+  }
+
+  /*Header*/
+  header {
+    width: auto;
+    justify-content: space-between;
+  }
+
+  nav {
+    display: flex;
+    margin-right: 20px;
+  }
+
+  nav a {
+    margin: 10px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: flex-end;
+    text-align: end;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  /*Big News*/
+  .container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .container img {
+    width: 70%;
+    margin-left: 40px;
+  }
+
+  .container div {
+    margin: 0 20px 0 20px;
+  }
+
+  /*Other News*/
+  .other-news {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .container-two {
+    position: relative;
+    width: 20%;
+    height: 348px;
+    margin: 12px;
+    border-radius: 5px;
+    box-shadow: 5px 5px 0 0 rgba(207, 73, 84, 0.705);
+  }
+
+  .container-two img {
+    height: 348px;
+    border-radius: 5px;
+  }
```

Comment:
Some of the images look a little distorted. Try `object-fit: cover;`to crop them and keep the correct ratio :)